### PR TITLE
Bastien/fib levels

### PR DESCRIPTION
**Pull request by bastienboutonnet**
- added fibo level calculations off the 24h price ticker. May make it more clever one day...
- untested cos I dunno how this app engine thingie works :p

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -156,7 +156,8 @@ def reply(msg=None, img=None):
                         pairs.append(pair)
                 r = 'Reply with /<assetpair> to get bid/ask prices\n{}'.format(', '.join(pairs))
                 reply(r)
-            elif text[1:].upper() in ASSETPAIRS.keys():
+
+            elif text.split(' ')[0][1:] in ASSETPAIRS.keys():
                 pair = text[1:].upper()
```
Comment (enzoblindow) [inline main.py]: pair also needs to be adjusted to read from the first part of the split `pair = text.split(' ')[0][1:]`

**Review by enzoblindow — CHANGES_REQUESTED** [review]
One small change, otherwise good 👍

File: main.py (hunk 1 above)
Comment (bastienboutonnet) [reply main.py]: well spotted man!

**Comment by bastienboutonnet** [comment]
last commit forgot to mention that Enzo's changes were taken into account

**Review by enzoblindow — APPROVED** [review]

**Pull request merged**